core/source/org/libreoffice/ide/eclipse/core/wizards/pages/UnoPackageExportPage.java

```diff
@@ -145,7 +145,9 @@ private void loadData() {
             i++;
         }
 
-        mContentSelector.loadDefaults();
+        if(selected!=false){
```

Comment:
You can shorten the `selected!=false` and write it as `selected`.
Also please keep some space around the braces as the other code does:

`if (selected) {`